csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -130,6 +130,9 @@ internal static class NativeMethods
         [DllImport(nativeLib, CharSet = charSet)]
         public static extern IntPtr /*(OrtStatus*)*/ OrtDisableSequentialExecution(IntPtr /*(OrtSessionOptions*)*/ options);
 
+        [DllImport(nativeLib, CharSet = charSet)]
+        public static extern IntPtr /*(OrtStatus*)*/ OrtSetOptimizedModelFilePath(IntPtr /* OrtSessionOptions* */ options, [MarshalAs(UnmanagedType.LPWStr)]string optimizedModelFilepath);
```

Comment:
default is BSTR right? why cant you use default? because string length could be longer?

Reply:
The Model uri for Model.Save function is a unicode backed wstring. Hence, need to use LPWStr to avoid string to be translated into a garbage string.